packages/jwt-middleware/README.md

```diff
@@ -9,23 +9,40 @@ This middleware allows to secure execution of a webtask using JWT tokens by vali
 ```
 {
     "@webtask/middleware-compiler": "1.3.0", 
-    "express-jwt": "5.3.1",
-    "jws": "3.1.4",
-    "jwks-rsa": "1.2.1"
+    "@webtask/jwt-middleware": "1.0.0"
 }
 ```
 
 2. Set the `wt-compiler` metadata property on your webtask to `@webtask/middleware-compiler`.
 
-3. Set the `wt-compiler` metadata property to `@webtask/jwt-middleware`.
+3. Set the `wt-middleware` metadata property to `@webtask/jwt-middleware`.
 
-4. Set the `wt-authorize-execution` metadata property to any value other than `0` to secure the webtask.  Value of `0` or missing `wt-authorize-execution` metadata property will result with the webtask execution being unsecure.
+4. Set the `wt-authorize-execution` metadata property to any value other than `0` to secure the webtask.  Value of `0` or missing `wt-authorize-execution` metadata property will result in the webtask execution being unsecure.
```

Comment:
I think that instead of using the word '*secure*', we could be a bit more explicit.

> ... to secure the webtask.

becomes

> ... to require either the `wt:owner:<container>` or `wt:admin` or the custom scope encoded in the `wt-execution-scope` metadata property.

Also:

> result in the webtask execution being unsecure.

becomes

> disable any authorization checks for the execution of the webtask.

These are just suggestions. I think it sends the wrong message to potential users that this might be 'unsecure' because it could be a perfectly valid use-case to use this for authentication and do custom authorization in your business logic based on the `req.user` value.

Reply:
@rwtombaugh Geoff is suggesting `in` instead of `with` (I changed it based on previous comments).  Thoughts?